jcl/src/openj9.gpu/share/classes/module-info.java

```diff
@@ -23,6 +23,10 @@
 
 /*[REM] This file must not use tabs because the dependency recognition code in openjdk does not support them. */
 
+/**
+ * Defines an API to perform certain operations using any connected CUDA capable GPU,
```

Comment:
I think we should remove 'an' for consistency with other modules.

Reply:
fixed